doc/radosgw/dynamicresharding.rst

```diff
@@ -14,17 +14,17 @@ online bucket resharding.
 
 Each bucket index shard can handle its entries efficiently up until
 reaching a certain threshold number of entries. If this threshold is
-exceeded the system can encounter performance issues. The dynamic
+exceeded the system can suffer from performance issues. The dynamic
```

Comment:
double entendre

Leave it in.

Reply:
@zdover23  I'm uncertain if you're suggesting that I leave this as `encounter`, or as `suffer from`.